### DISCUSSION TITLE
Make use of existing music visualizer

### DISCUSSION THREAD

**Discussion by LimitlessGreen** [EnviralDesign/GeoPix#28] [Ideas]
First, this shouldn't have a high priority, and it's more something for a more distant future.

Specially for LED walls, a visualizer node (using existing tools like Goom or ProjectM) could be helpful for quick but bpm synced effects.

**Comment by EnviralDesign** [comment]
great idea - if I could find a way to build/import these as shaders or c++ top I could easily add it in as an IO node. Touch excels at this type of thing as well natively with feedback tops n such.

I honestly have plans to really buff out the content of GeoPix in the IO tab greatly once features settle and everything is functional the way it needs to be, creating good content is still a somewhat tedious process with ramps and noise only taking one so far.

**Reply by LimitlessGreen** [reply]
ProjectM is basically a C++ library: https://github.com/projectM-visualizer/projectm/tree/master/src/libprojectM. It's reimplementing MilkDrop (WinAmp times) with tons of examples (honestly, a lot of them look a bit ... old).

Step by step :)
Yeah, in comparison with other node editors there is a bit to catch up. But I mean...it's usable :)
